XamlControlsGallery/ControlPagesSampleCode/TeachingTip/TeachingTipSample1_xaml.txt

```diff
@@ -0,0 +1,10 @@
+﻿<Button Content="Show TeachingTip" Click="TestButtonClick1" />
+
+<controls:TeachingTip x:Name="ToggleThemeTeachingTip1"                   
+	Target="{x:Bind ThemeButton}"
+	Title="Change themes without hassle"                                        
```

Comment:
should the above button be named?

Reply:
Naming doesn't hurt but doesn't seem necessary either.

Reply:
I didn't add a name to the button because it did not impact the code being shown but I am happy to add it in the spirit of good practice. Which is preferred here?

Reply:
If the name isn't required, we shouldn't name it. Naming an element is particularly useful when referencing it in codebehind, but since we aren't referencing this button anywhere else, adding a name would be a false hint that it might be used in codebehind. We also don't have a pattern that names every element, so I think we shouldn't name it in this case.

Reply:
Spoke to @stmoy and he advised not adding a name to the button if it is not used - which it is not here. I will mark this resolved for now, but please feel encouraged to unresolve this if we should discuss it more.